### PR TITLE
[TT-3276] Do not panic when valu type nil

### PR DESCRIPTION
**Pull request by furkansenharputlu**

### CODE REVIEW

**Review by vaske — APPROVED** [review]

File: schema/field.go [hunk 1]
```diff
@@ -941,7 +941,7 @@ func (field *Field) setupValuerAndSetter() {
 						case *sql.RawBytes:
 							bytes = []byte(*t)
 						default:
-							panic("wrong type")
+							return
```
Comment (buger) [inline schema/field.go:944]: It is too generic. 

Plus I do not get how (type) of the object can be nil? 
Why it happens for OAS PR?

Comment (furkansenharputlu) [reply schema/field.go:944]: Because in `openapi3.Swagger` struct, there are struct pointers like `*Contact`, `*License` etc. they can be nil.

Comment (buger) [reply schema/field.go:944]: I see.

OK, it can work, I guess.

Comment (buger) [reply schema/field.go:944]: Can we then add more specific check for Nil case?

Comment (furkansenharputlu) [reply schema/field.go:944]: @buger I added a `case nil:`. Is that ok?

**Comment by buger** [comment]
Looks good!

**Pull request merged**